src/shared/components/ncTable/sections/CustomTable.vue

```diff
@@ -287,7 +358,64 @@ export default {
 }
 </script>
 
+<style>
+.vs__dropdown-menu {
+	min-width: 95px !important;
+}
+</style>
+
 <style lang="scss" scoped>
+
+:deep(.vs__clear) {
+	display: none;
+}
+
+:deep(.v-select) {
+	min-width: 95px !important;
+	.vs__dropdown-toggle {
+		background: none;
+	}
+}
+
+.selected-page{
+	padding-left: 5px;
+
+	display:inline-flex;
+	align-items: center;
+}
+
+.page-number{
+	padding-inline: 5px;
+}
+
+.large-width{
+	width: 100vw !important;
+	left: 0 !important;
+}
+
+.pagination-items{
+	background-color: var(--color-main-background);
+	border-radius: var(--border-radius-large);
+	pointer-events: all;
+
+	display: flex;
+	align-items: center;
+}
+
+.pagination-footer{
+	position:absolute;
```

Comment:
When testing with applications where multiple tables are rendered on the same page the pagination items overlap each other, my suggestion would be to use sticky positioning here and give the parent container a relative positioning.